Makefile

```diff
@@ -1,32 +1,10 @@
-
 .PHONY: build clean test
 
 build:
-	jbuilder build @install
+	dune build
 
 test:
-	jbuilder runtest
-
-install:
-	jbuilder install
-
-uninstall:
-	jbuilder uninstall
+	dune runtest
 
 clean:
-	rm -rf _build *.install
-
-REPO=../../mirage/opam-repository
-PACKAGES=$(REPO)/packages
-# until we have https://github.com/ocaml/opam-publish/issues/38
-pkg-%:
-	topkg opam pkg -n $*
-	mkdir -p $(PACKAGES)/$*
-	cp -r _build/$*.* $(PACKAGES)/$*/
-	cd $(PACKAGES) && git add $*
-
-PKGS=$(basename $(wildcard *.opam))
-opam-pkg:
-	$(MAKE) $(PKGS:%=pkg-%)
```

Comment:
Out of curiosity, what's the recommended way of creating+publishing packages?

Reply:
you can use [dune-release](https://github.com/samoht/dune-release). That's not perfect but does relatively well in general :-) Feel free to open issues if that's not the case.